### ISSUE TITLE
'redis.Cmder.String()' Method returns data that is too long to be readable

### ISSUE THREAD

**Issue by hirampeng** [go-redis/redis#1507]
source:
```
func appendUTF8String(dst []byte, src []byte) []byte {
	if isSimple(src) {
		dst = append(dst, src...)
		return dst
	}

	s := len(dst)
	dst = append(dst, make([]byte, hex.EncodedLen(len(src)))...)
	hex.Encode(dst[s:], src)
	return dst
}
```

route:
redis.Cmder->String()->cmdString()->internal.AppendArg(b, arg)->appendUTF8String(b, Bytes(v))

describe:
The result of this method(hex.Encode) is that the data returned is long and not what I expected

example:
```
cmd:
get hiram:peng
result:
get 686972616d3a70656e67: redis: nil
except:
get hiram:peng: redis: nil
```

other:
I expect the interface 'redis.cmder' to have the Val method fetch the exact value returned

**Comment by hirampeng** [comment]
solution:
    location:
    go_library/pkg/mod/github.com/go-redis/redis/v8@v8.1.3/internal/arg.go:79
    add line `':': true`,

**Comment by vmihailenco** [comment]
Should be fixed in v8.2.2
